app/channel.go

```diff
@@ -755,6 +755,11 @@ func (a *App) UpdateChannelMemberSchemeRoles(channelId string, userId string, is
 		return nil, err
 	}
 
+	// Notify the clients that the member notify props changed
+	evt := model.NewWebSocketEvent(model.WEBSOCKET_EVENT_CHANNEL_MEMBER_UPDATED, "", "", userId, nil)
```

Comment:
The term used in most parts of the code is `message` instead of `evt`. Would you mind changing the variable name?

Reply:
Done! Thanks